src/rx/main.c

```diff
@@ -2,18 +2,31 @@
 #include "drivers/i2c.h"
 #include "drivers/gpio.h"
 #include "drivers/pca9685.h"
+#include "drivers/can.h"
 
 int main(void)
 {
     gpio_output_mode_init();
     gpio_i2c_mode_init();
     i2c1_init();
+    gpio_can_mode_init();
     pca9685_init(PCA9685_DEFAULT_FREQUENCY);
+    
+    if (CAN_InitStatus_Success == can_init()) {
```

Comment:
Maybe we will use the same sequence of operands as [here](https://github.com/lamia-mortis/led-headlights/blob/main/src/tx/main.c#L14) during comparing operation?